cost/azure/idle_compute_instances/azure_idle_compute_instances.pt

```diff
@@ -195,10 +205,10 @@ policy "azure_resource_policy" do
     detail_template <<-EOS
 # Azure idle compute instances
 
-| Region/Location | Name | CPU Average | Resource Group | Tags |
-| -------- | ---- | ----------- | -------------- | ---- |
+| Subscription Name | Region/Location | Name | CPU Average | Resource Group | Tags |
```

Comment:
please sort by Subscription name, then region

Reply:
Sorted by subscriptionName and location